python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+import warnings
```

Comment:
This is an interesting thing: my first reaction was to ask to use cuml.logger for this, but it is more pythonic to use a `DeprecationWarning` so I think it is fine, I think in general we should codify that using the Python warning is the preferred way of doing deprecations.

Reply:
Sounds good. I'll file an issue and reference this comment